app/views/forms/submission_email/submission_email_confirmed.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1 class="govuk-heading-l">
+  <span class="govuk-caption-l"><%= @submission_email_form.form.name %></span>
+  <%= t("page_titles.confirm_email_success") %>
+</h1>
+
+<%= t('email_code_success.body_html', submission_email: @submission_email_form.form.submission_email) %>
+
+<%= govuk_link_to(t('email_code_success.continue'), form_path) %>
```

Comment:
We need to wrap `<p>` around this link otherwise the font size of it will be smaller than the text above.